### PR TITLE
Delete previous folder, ignore error if not exists

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:

Instead of checking whether the folder exists we just try to delete it
recursively while ignoring the errors instead, as we do only care to
delete it if it exists in order to remove previously created artifacts.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes issues like these: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/push-release-kubevirt-tag/1384121154242875392#1:build-log.txt%3A1674

**Special notes for your reviewer**:
/cc @rmohr @davidvossel 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: automation/release.sh [hunk 1]
```diff
@@ -72,9 +72,7 @@ function update_github_release() {
 
 function upload_testing_manifests() {
     # replaces periodic-kubevirt-update-release-x.y-testing-manifests periodics
-    if [ "$(gsutil ls 'gs://kubevirt-prow/devel/release/kubevirt/kubevirt/'|grep $DOCKER_TAG|wc -l)" -gt 0 ]; then
```
Comment (rmohr) [inline automation/release.sh:75]: Doid you consider :https://cloud.google.com/storage/docs/gsutil/commands/stat?

Seems like `gsutil -q stat gs://some-bucket/some-subdir/` seems to be extra there for scripting.

Comment (dhiller) [reply automation/release.sh:75]: I remember having considered using it a while ago, don't remember why I didn't. Will look into this.

Comment (dhiller) [reply automation/release.sh:75]: Now I remember: it exits with 1 for non existing objects. Therefore I initially went with the `gsutil ls ... | grep` approach. So if we use stat we would also have to ignore errors, which IMHO makes no difference to just trying to delete it blindly. WDYT?

Comment (rmohr) [reply automation/release.sh:75]: ack

**Pull request merged**